### DISCUSSION TITLE
URL Encoding issues?

### DISCUSSION THREAD

**Discussion by pluja** [casdoor/casdoor#1898] [Q&A] [answered]
I have setup Casdoor and I'm trying the login flow with Immich. I have added the Callback urls as stated in the Immich docs:
https://immich.app/docs/administration/oauth

But Casdoor is saying:

```
There was a problem signing you in..
Redirect URI: https%3A%2F%2Fimmich.mydomain.com%2Fauth%2Flogin doesn't exist in the allowed Redirect URI list
```

Although the URL: https://immich.mydomail.com/auth/login is added.

**Comment by hsluoyz** [comment, marked as answer]
@pluja already fixed in: https://github.com/casdoor/casdoor/commit/c57c6e37dd25e448b1d266acce1f39faa420f942 , use a version >= v1.323.1
